Quotient/events/serveraclevent.h

```diff
@@ -0,0 +1,19 @@
+// SPDX-FileCopyrightText: 2024 James Graham <james.h.graham@protonmail.com>
+// SPDX-License-Identifier: LGPL-2.1-or-later
+
+#pragma once
+
+#include "stateevent.h"
+
+namespace Quotient {
+class QUOTIENT_API ServerAclEvent : public StateEvent {
+public:
+    QUO_EVENT(ServerAclEvent, "m.room.server_acl")
```

Comment:
```suggestion
class QUOTIENT_API RoomServerAclEvent : public StateEvent {
public:
    QUO_EVENT(RoomServerAclEvent, "m.room.server_acl")
```
To follow the naming convention for (most of) other room events. Whether to rename the file is up to you but if I were to keep it (see above) I would rename it too.

Reply:
Updated